packages/jade-coffee/package.js

```diff
@@ -1,19 +1,21 @@
 Package.describe({
   summary: "Jade template language with inline coffeescript and Javascript support",
-  version: "0.1.4",
+  version: "0.1.6",
   name: "xiphy:jade-coffee",
   git: "https://github.com/xiphias/meteor-jade-coffee.git",
   documentation: "../../README.md"
 });
 
 Package.registerBuildPlugin({
-  name: "compileJade",
+  name: "compileJadeBatch",
```

Comment:
Aha, I see this is not a function name..anyways it doesn't compile in Batch yet, could you revert to the old name? (compileJade)